### PR TITLE
[FA] - Ch3 - P3.1 and P3.2

### PR DESCRIPTION
**Pull request by kambizG**
Hi @jowharshamshiri jan and @hamedonline jan,

I'm done with p3-ch3. Can anyone help with the review?

And, here is a list of new phrases for G, which I'll add later if we agreed, after the review and before merging. 

Part 3
| string | متغییر متنی |
| learning rate | نرخ یادگیری |
| initialize | مقداردهی اولیه |
| scheduler | زمانبند |
| decaying | محو شونده |
| annealing | بازپختی |
| upload | بارگذاری |
| hub | هاب |
| fit | فیت |
| hyperparameter | پارامتر سطح بالا |
| evaluate | ارزیابی کردن|

### CODE REVIEW

**Review by lewtun — APPROVED** [review]
Hi @kambizG thank you so much for continuing the Persian translations! I can't comment on the text, but I checked the rendering on the website and that looks good to me.

Happy to merge when I get the green light from the other reviewers :)

**Comment by lewtun** [comment]
Gently pinging @jowharshamshiri and @hamedonline in case they have time to take a look at this 🤗 

No worries if you're both busy - I can merge this and we can iterate on whatever is needed in separate PRs :)

**Comment by hamedonline** [comment]
> Hi @jowharshamshiri jan and @hamedonline jan,
> I'm done with p3-ch3. Can anyone help with the review?

Thanks a lot dear @kambizG for translations.

> Gently pinging @jowharshamshiri and @hamedonline in case they have time to take a look at this 🤗
> No worries if you're both busy - I can merge this and we can iterate on whatever is needed in separate PRs :)

About the review, I believe I'll be able to do a quick one by the end of this weekend dear @lewtun. Sorry for the delay.

**Comment by hamedonline** [comment]
Dear @kambizG,
I'm so sorry it took this long for the review; a series of unfortunate events occurred one after another, making it impossible for me to get focused on your great contribution. I'll be going over translations line by line in coming days and post a final comment indicating when the whole review is finished.

**Comment by lewtun** [comment]
Hi folks, shall we go ahead and merge this translation in its current form? I think it's totally fine if we need to make corrections in separate PRs - by merging this, we enable the Persian community to already benefit from @kambizG's work 🤗

**Comment by hamedonline** [comment]
> Hi folks, shall we go ahead and merge this translation in its current form? I think it's totally fine if we need to make corrections in separate PRs - by merging this, we enable the Persian community to already benefit from @kambizG's work 🤗

Dear @lewtun, I've already finished reviewing 2 out of 3 files with notes in this PR. They are not visible since the review is not submitted yet (due in a day). I believe applying changes should not take more than a couple of days and we'll have another nice one added to Persian translations thanks to @kambizG.

**Comment by lewtun** [comment]
> Dear @lewtun, I've already finished reviewing 2 out of 3 files with notes in this PR. They are not visible since the review is not submitted yet (due in a day). I believe applying changes should not take more than a couple of days and we'll have another nice one added to Persian translations thanks to @kambizG.

Sounds great! Thank you so much for taking such care with the translations 🙏
